docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -22,233 +22,91 @@ available, go to <<alerting-getting-started>>.
-custom action interval.
-+
---
-[[alerting-concepts-suppressing-duplicate-notifications]]
-[TIP]
-==============================================
-Since actions are triggered per alert, a rule can end up generating a large
-number of actions. Take the following example where a rule is monitoring three
-servers every minute for CPU usage > 0.9, and the rule is set to notify
-`On check intervals`:
-
-* Minute 1: server X123 > 0.9. _One email_ is sent for server X123.
-* Minute 2: X123 and Y456 > 0.9. _Two emails_ are sent, one for X123 and one for Y456.
-* Minute 3: X123, Y456, Z789 > 0.9. _Three emails_ are sent, one for each of X123, Y456, Z789.
-
-In this example, three emails are sent for server X123 in the span of 3 minutes
-for the same rule. Often, it's desirable to suppress these re-notifications. If
-you set the rule notify setting to `On custom action intervals` with an interval
-of 5 minutes, you reduce noise by getting emails only every 5 minutes for
-servers that continue to exceed the threshold:
-
-* Minute 1: server X123 > 0.9. _One email_ is sent for server X123.
-* Minute 2: X123 and Y456 > 0.9. _One email_ is sent for Y456.
-* Minute 3: X123, Y456, Z789 > 0.9. _One email_ is sent for Z789.
-
-To get notified only once when a server exceeds the threshold, you can set the
-rule notify setting to `On status changes`. 
-==============================================
---
-
-[role="screenshot"]
-image::images/rule-flyout-general-details.png[alt='All rules have name, tags, check every, and notify properties in common']
-
 [float]
 [[defining-rules-type-conditions]]
 ==== Rule type and conditions
 
-Depending upon the {kib} app and context, you might be prompted to choose the type of rule to create. Some apps will preselect the type of rule for you.
+Depending on the {kib} app and context, you might be prompted to choose the type of rule to create. Some apps will preselect the type of rule for you.
+
+Each rule type provides its own way of defining the conditions to detect, but an expression formed by a series of clauses is a common pattern. For example, in an index threshold rule, the `WHEN` clause enables you to select an aggregation operation to apply to a numeric field.
 
 [role="screenshot"]
-image::images/rule-flyout-rule-type-selection.png[Choosing the type of rule to create]
+image::images/rule-flyout-rule-conditions.png[UI for defining rule conditions on an index threshold rule,500]
 
-Each rule type provides its own way of defining the conditions to detect, but an expression formed by a series of clauses is a common pattern. Each clause has a UI control that allows you to define the clause. For example, in an index threshold rule, the `WHEN` clause allows you to select an aggregation operation to apply to a numeric field.
+All rules must have a check interval, which defines how often to evaluate the rule conditions. Checks are queued; they run as close to the defined value as capacity allows.
```

Comment:
Do you think a screenshot of check every would be useful?

Reply:
It's actually in the screenshot that appears above this paragraph. It's unfortunate that the label is cut off due to the screenshot window size, but it's there.